presto-main/src/test/java/com/facebook/presto/operator/scalar/TestHyperLogLogFunctions.java

```diff
@@ -140,6 +140,19 @@ public void testCardinalityManyColumns()
         functionAssertions.assertFunctionWithError(projection, BIGINT, uniqueElements, error);
     }
 
+    @Test
+    public void testNoisyCardinalitySingleColumn()
+    {
+        long uniqueElements = 10000;
+        double error = uniqueElements * 0.05;
+        HyperLogLog hll = HyperLogLog.newInstance(NUMBER_OF_BUCKETS);
```

Comment:
any other test cases that can be added like corner cases etc? For example, for different number of unique elements?